NuGetCleaner/Program.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using System.Reflection;
+using System.IO;
+using CommandLine;
+using System.ComponentModel;
+using NuGet.Configuration;
+
+namespace NuGetCleaner
+{
+    class Program
+    {
+
+        public static void Main(string[] args)
+        {
+            var osNameAndVersion = System.Runtime.InteropServices.RuntimeInformation.OSDescription;
+            if (!osNameAndVersion.Contains("Windows"))
```

Comment:
You should be able to do a more concrete "is windows" check with this:
https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.runtimeinformation.isosplatform?view=netframework-4.8

Reply:
Consider: 
```cs
private static IsWindows(){
            if (System.Runtime.InteropServices.RuntimeInformation.IsOSPlatform(System.Runtime.InteropServices.OSPlatform.Windows))
            {
                return true;
            }

            return false;
}
```